README.md

```diff
@@ -245,6 +246,30 @@ with:
   web-identity-token-file: /var/run/secrets/eks.amazonaws.com/serviceaccount/token
 ```
 
+### Proxy Configuration
+
+If you run in self-hosted environments and in secured environment where you need use a specific proxy you can it in the action manually.
```

Comment:
> If you run in self-hosted environments and in secured environment where you need use a specific proxy you can it in the action manually.

I think you missed a word here. Change to something like

> If you run in self-hosted environments and in secured environment where you need use a specific proxy you can set it in the action manually.

Reply:
done